libs/resiliency/include/hpx/resiliency/async_replicate.hpp

```diff
@@ -186,6 +289,24 @@ namespace hpx { namespace resiliency { namespace experimental {
             std::forward<F>(f), std::forward<Ts>(ts)...);
     }
 
+    ///////////////////////////////////////////////////////////////////////////
+    // Asynchronously launch given function \a f exactly \a n times on \a n
+    // different localities, where \a n is the size of the vector of \a ids.
+    // Verify the result of those invocations using the given predicate \a pred.
+    // Run all the valid results against a user provided voting function.
+    // Return the valid output.
+    template <typename Pred, typename Action, typename... Ts>
+    hpx::future<typename hpx::util::detail::invoke_deferred_result<Action,
+        hpx::naming::id_type, Ts...>::type>
+    tag_invoke(async_replicate_validate_t,
+        std::vector<hpx::naming::id_type> ids, Pred&& pred, Action&& action,
```

Comment:
If we decided to keep the `vector<id_type>` it should be passed by `const&` to avoid copying.

Reply:
Right. Forgot to make them const&.